.pipelines/pipeline.user.windows.buddy.yml

```diff
@@ -57,8 +57,13 @@ test:
     - !!testcommand
       name: 'NPM Test'
       command: 'npmTest.cmd'
-      fail_on_stderr: true
+      fail_on_stderr: false
       skip_on_developer_box: true
+      testresults:                    # RECOMMENDED: If you want CDPX to collect any test results files and have the results be 
```

Comment:
whats CDPX stand for?

Reply:
[Cross-Platform Cloud Delivery Pipeline](https://onebranch.visualstudio.com/Pipeline/_wiki/wikis/Pipeline.wiki/330/OneBranch-Pipeline-General?anchor=%5Bthe-cross-platform-cloud-delivery-pipeline-(cdpx)%5D(%2Fcdpx-%252d-cross%252dplatform-cloud-delivery-pipeline))  its the internal org that does all the build/deployment architecture